wal/src/manager.rs

```diff
@@ -286,44 +278,41 @@ pub trait WalManager: Send + Sync + fmt::Debug + 'static {
     async fn scan(&self, ctx: &ScanContext, req: &ScanRequest) -> Result<BatchLogIteratorAdapter>;
 }
 
-/// Adapter to convert a blocking interator to a batch async iterator.
+#[derive(Debug)]
+pub enum InnerIterator {
```

Comment:
How about naming it `BatchLogIterator`?

Reply:
But the inner's return is not a batch, but is just a single log entry.
I propose to rename it to `LogIterator`.

Reply:
Done.